.github/workflows/automate_changeset_feedback.yml

```diff
@@ -26,70 +26,15 @@ jobs:
-
-        # We avoid using the in-source script since this workflow has elevated permissions that we don't want to expose
-      - name: Generate Feedback
-        id: generate-feedback
-        run: |
-          rm -f generate.js
-          wget -O generate.js https://raw.githubusercontent.com/backstage/backstage/master/scripts/generate-changeset-feedback.js 1>&2
-          node generate.js FETCH_HEAD > feedback.txt
-
-      - name: Post Feedback
-        uses: actions/github-script@v6
-        env:
-          ISSUE_NUMBER: ${{ github.event.pull_request.number }}
+      - uses: backstage/actions/changeset-feedback@v0.5.12
+        name: Generate feedback
         with:
-          script: |
-            const owner = "backstage";
-            const repo = "backstage";
-            const marker = "<!-- changeset-feedback -->";
-            const feedback = require('fs').readFileSync('feedback.txt', 'utf8');
-            const issue_number = Number(process.env.ISSUE_NUMBER);
-            const body = feedback.trim() ? feedback + marker : undefined
-
-            const existingComments = await github.paginate(github.rest.issues.listComments, {
-              owner,
-              repo,
-              issue_number,
-            });
-
-            const existingComment = existingComments.find((c) =>
-              c.user.login === "github-actions[bot]" &&
-              c.body.includes(marker)
-            );
-
-            if (existingComment) {
-              if (body) {
-                if (existingComment.body !== body) {
-                  console.log(`updating existing comment in #${issue_number}`);
-                  await github.rest.issues.updateComment({
-                    owner,
-                    repo,
-                    comment_id: existingComment.id,
-                    body,
-                  });
-                } else {
-                  console.log(`skipped update of identical comment in #${issue_number}`);
-                }
-              } else {
-                console.log(`removing comment from #${issue_number}`);
-                await github.rest.issues.deleteComment({
-                  owner,
-                  repo,
-                  comment_id: existingComment.id,
-                  body,
-                });
-              }
-            } else if (body) {
-              console.log(`creating comment for #${issue_number}`);
-              await github.rest.issues.createComment({
-                owner,
-                repo,
-                issue_number,
-                body,
-              });
-            }
+          diffRef: 'origin/master'
+          marker: <!-- changeset-feedback -->
+          issue-number: ${{ github.event.pull_request.number }}
+          botUsername: github-actions[bot]
+          app-id: ${{ secrets.BACKSTAGE_GOALIE_APPLICATION_ID }}
+          private-key: ${{ secrets.BACKSTAGE_GOALIE_PRIVATE_KEY }}
+          installation-id: ${{ secrets.BACKSTAGE_GOALIE_INSTALLATION_ID }}
```

Comment:
This mixture of casing is a bit unfortunate. We should probably unify to all use camelCase eventually.

Reply:
Yep although other way around, action inputs generally use `kebab-case`

Reply:
Good point! I'll put on my TODO.

Reply:
@freben I am also okay with parking this PR till we get the camel casing right after holidays. No high prio on this anyway. Let me know!

Reply:
I want it in now! :)

Reply:
Will press merge once the release is out